routes/v1/users.js

```diff
@@ -34,6 +34,36 @@ module.exports = function(/*middleware*/) {
 		});
 	});
 
+    app.put('/:uid/password', apiMiddleware.requireUser, apiMiddleware.exposeAdmin, function(req, res) {
+        if (!utils.checkRequired(['currentPassword', 'newPassword'], req, res)) {
+            return false;
+        }
+
+        var uid = parseInt(req.params.uid, 10);
+        if (uid !== parseInt(req.user.uid, 10) && !res.locals.isAdmin) {
+            return errorHandler.respond(401, res);
+        }
+        var data = {
+            uid: uid,
+            currentPassword: req.body.currentPassword,
+            newPassword: req.body.newPassword
+        };
+        Users.changePassword(uid, data, function(err) {
+            return errorHandler.handle(err, res);
+        });
+    });
+
+    app.put('/:uid/password_reset', apiMiddleware.requireUser, apiMiddleware.requireAdmin, function(req, res) {
+        if (!utils.checkRequired(['newPassword'], req, res)) {
+            return false;
+        }
+
+        var uid = parseInt(req.params.uid, 10);
+        Users.resetPassword(uid, req.body.newPassword, function(err) {
```

Comment:
Well, this shouldn't go in like this if it does nothing.
Can you please remove this, and if you send a pull request to core with this method, it would probably be a good idea to use it here as well.